### DISCUSSION TITLE
Model Reliability v.s. Model Robustness

### DISCUSSION THREAD

**Discussion by Jingkang50** [Jingkang50/OODSurvey#11] [General]
Some readers asked me about the difference of these two terms.
Here is my thought.

**Model Reliability:**
Reliability modeling is the process of predicting or understanding the reliability of a component or system prior to its implementation. [[ref]](https://en.wikipedia.org/wiki/Reliability_engineering#:~:text=Reliability%20modeling%20is%20the%20process,system%20prior%20to%20its%20implementation.)

**Model Robustness:**
A model is robust when the accuracy does not change significantly from the baseline accuracy under various conditions. [[ref]](https://thirdeyedata.io/robustness-measurement-of-machine-learning-models-with-examples-in-python/#:~:text=Github%20repository%20avenir.-,Model%20Robustness,e.g%20it%20includes%20human%20bias.)

So in my opinion, OOD detection is pursuing model reliability, as the models are required to estimate the probability whether the given sample is ID. OOD detection is all about **confidence**. A good example of model robustness is OOD generalization, which requires stable performance even with domain shifts. OOD generalization is all about **accuracy**.

In sum:
OOD detection -> Model Reliability
OOD generalization -> Model Robustness

**Comment by XixiLiu95** [comment]
Calibration is also one way to evaluate whether the model is reliable or not.
